### ISSUE TITLE
How to disable playpen feature?

### ISSUE THREAD

**Issue by killercup** [rust-lang/mdBook#747]
I'm currently writing a book on CLI applications in Rust (<https://github.com/rust-lang-nursery/cli-wg/pull/47>). Executing the code snippets in playpen will sadly not be very useful.

Is there a way to disable the playpen feature globally?

**Comment by mattico** [comment]
Apparently not! This should be easy to fix.

**Comment by badboy** [comment]
This should be covered by https://github.com/rust-lang-nursery/mdBook/pull/744:

```
```rust,noplaypen
fn main(){
    println!("Don't run it.");
}
\```
```

**Comment by ehuss** [comment]
`noplaypen` needs documentation.

Also, @killercup, is that good enough?

**Comment by detly** [comment]
Could this be a global thing, set in `book.toml`? I had hoped that eg. `[output.html.playground]` would let me control it, but alas no.

**Comment by pineapplehunter** [comment]
I thought a global flag in `book.toml` would be nice as @detly said. So I tried implementing in a draft PR #1546. Would this help?

**Comment by adpaco-aws** [comment]
@pineapplehunter a global flag would be helpful. Does #1546 work? If it does, why not mark as ready for review?

**Comment by schungx** [comment]
Yes, I keep having to put `no_run` in all my code examples just to disable the playpen.

**Comment by pineapplehunter** [comment]
@adpaco-aws Thanks.
I'll check my code again and mark it as ready for review.

**Comment by ehuss** [comment]
Closing as resolved by #1546.
